inspector/src/components/actionTabs/actionTabs.scss

```diff
@@ -294,7 +294,7 @@ $line-padding-left: 2px;
                     .file-upload {
                         background: #222222;
                         border: 1px solid rgb(51, 122, 183);
-                        margin: 0px 10px;
+                        margin: 5px 10px 5px 10px;

```

Comment:
Why this change?

Reply:
Now (before PR):

![UI inspector](https://user-images.githubusercontent.com/20560340/70072061-925e1980-15f6-11ea-8216-d32dbd099ad0.png)

With the 5px vertical margin:
![UI inspector new](https://user-images.githubusercontent.com/20560340/70072286-f8e33780-15f6-11ea-863e-a7cdfedf31a4.png)

The margins look more consistent top and bottom with the 5px.
The file input component is not often used, if not at all so that may explain why it stayed like this.

Reply:
This way, the margin: 5px 10px 5px 10px; is the same as the "button" one.